cli/js/lib.deno.ns.d.ts

```diff
@@ -611,8 +602,8 @@ declare namespace Deno {
 
   /** Synchronously read from a resource ID (`rid`) into an array buffer (`buffer`).
    *
-   * Returns either the number of bytes read during the operation or End Of File
-   * (`Symbol(EOF)`) if there was nothing to read.
+   * Returns either the number of bytes read during the operation or `null` if
+   * there was nothing to read.
```

Comment:
EOF is still a concept, even if we aren't using a symbol. It's important that the documentation is specific here:


> Returns either the number of bytes read during the operation or End Of File (`null`) if there was nothing to read.

Reply:
We should comment that it is possible for `read()` to return successfully with `0` bytes and that it does not indicate EOF.